src/style/app.module.css

```diff
@@ -7433,3 +7436,51 @@ button[data-testid='createPostBtn'] {
   width: 2rem;
   border-width: 0.2rem;
 }
+
+.organizationCard_avatar {
+  border-radius: 50%;
+  background-blend-mode: darken;
+  background-color: var(--primary-bg-color);
+  height: 65px;
+  width: 65px;
+}
+
+.organizationCard_warning {
+  color: var(--warning-text-color);
+}
+
+.box :hover {
+  color: #b85c00; /* Example of a darker, accessible orange */
+}
+
+.organizationCard_header {
+  display: flex;
+  flex-direction: row;
+  padding-bottom: 10px;
+  padding-top: 10px;
+}
+
+.organizationCard_content {
+  padding-left: 20px;
+  padding-top: 10px;
+}
+
+.second_box > h4 {
+  font-size: 10px;
+  font-weight: bold;
+  text-decoration: none;
+  color: var(--primary-text-color);
+}
+
+.second_box > h5 {
+  text-decoration: none;
+  font-size: 10px;
+  font-weight: 100;
+  color: var(--gray-white);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`